hashing/main.cpp

```diff
@@ -158,13 +158,19 @@ static int example(mbedtls_platform_context* ctx)
 
 int main() {
     mbedtls_platform_context platform_ctx;
-    if(mbedtls_platform_setup(&platform_ctx) != 0) {
-        return -1;
+    int exit_code = MBEDTLS_EXIT_FAILURE;
```

Comment:
I think this function will always return MBEDTLS_EXIT_FAILURE, a failure code...